### PR TITLE
Features/376 replace download and donate button

### LINKED ISSUE #376: Replace 'Donate & Download' button
Following #362 we found out we can't add UTM tags to custom FRU elements. 

So let's bring back our old download button, and open FRU the same way we do with the donate links.

### PR DESCRIPTION
**Pull request by MelissaAutumn**
Resolves #376 

A bit of feature creep in this one. While re-working some things, I came to conclusion that I was over-complicating the way we load the donation form. So now it just replaces the links on startup, and this fixes any silly issue we've had with scrolling, page refreshing, etc...

**Note: We will have to confirm the utm tags are working correctly after this goes on staging.** 

Now onto the main show:

Re-implemented our button, and added an animation similar to the one FRU uses. It's like the no-name version, there's no increasing pulses on hover (because css animation hates changing duration mid animation cycle.) but it works, and allows us to hopefully add utm tags. FRU seems to remove the utm tags from the url, but they appear in the javascript event details. 

I've also added my prefer-reduce-motion change, which removes the animation. This still applies the background colour change.

Before Animation:
image
Mid Animation:
image
Hovered:
image

Tested this on Firefox, Edge, Safari, Chrome.

Edit: Hmm the heart animation is kind of super ugly on Firefox with some monitors. I'll look into it.

### CODE REVIEW

**Comment by Sancus** [comment]
Good job! I think the animation is great. However, the contents of the button need a little work. The text is quite a bit smaller than FRU's(which makes it contrast worse with the 'no thanks') and it's not vertically centered.

**Review by Sancus — CHANGES_REQUESTED** [review]
Good job! I think the animation is great. However, the contents of the button need a little work. The text is quite a bit smaller than FRU's(which makes it contrast worse with the 'no thanks') and it's not vertically centered.

**Comment by Alecaddd** [comment]
The contrast of the hover button is also not great.
Sorry to jump in, I'm gonna add myself as a reviewer and suggest some CSS fixes

**Comment by Sancus** [comment]
Yea that's a good point, the font color on hover should change like FRU's does.

**Comment by MelissaAutumn** [comment]
image
image

Sounds good, I've adjusted positioning to match staging more closely, and lightened the alpha level of the background-colour hover effect. Let me know I should swap the background-colour hover to the same green as the text.

**Review by Alecaddd — APPROVED** [review]
Just a few nits, for the rest looks good.
Great work!

File: assets/less/components/donation-elements.less [hunk 1]
```diff
@@ -53,8 +53,118 @@
...
+        .colour-blue(0.35);
+    }
+    35% {
+        .colour-blue(0.15);
+    }
+    to {
+        transform: scale(2.0);
+        .colour-blue(0.0);
+    }
 }
```
[the remark this answers is not in the document]
Comment (MelissaAutumn) [reply assets/less/components/donation-elements.less:180]: I don't think we do, I'll make a note to add one in the near future.

**Comment by MelissaAutumn** [comment]
Gonna have to squish these commits on merge 😅 

I've tweaked the size, positioning and animation slightly. It's faster than FRU, but not as fast as their hover speed-up. So like a happy medium?

**Review by Sancus — APPROVED** [review]

**Pull request merged**